pr2_guide/Spot spot
---
bool ret
